Ironmon-Tracker.lua

```diff
@@ -418,6 +420,29 @@ function Main.GenerateNextRom()
 	}
 end
 
+function Main.SaveCurrentRom(filename)
+	if Main.FileExists(filename) then
+		Main.CopyFile(filename, string.format("prev-seed %s", filename))
+		Main.CopyFile(string.format("%s.log", filename), string.format("prev-seed %s.log", filename))
+	end
+end
```

Comment:
Could you move the check to see the "file I wish to copy" exists or not inside of the function that performs the copy? This way any other devs who may want to use the `CopyFile()` function don't have to implicitly understand they need to first check to verify the file exists.

Reply:
[e8672f6](https://github.com/besteon/Ironmon-Tracker/pull/283/commits/e8672f63862017bae44eaeee54cb9b15e809ae64)